src/components/NavBar.vue

```diff
@@ -46,37 +29,45 @@
       />
       <v-spacer />
 
-      <nav-button
+      <amb-button
+        with-badge
+        data-cy="timeline-icon"
+        is-icon
+        icon="history"
+        btn-color="secondary"
+        to="timeline"
+      />
+
+      <amb-button
+        with-badge
+        data-cy="download-off"
+        is-icon
+        icon="download-off"
+        btn-color="secondary"
+        v-if="!isEdgeConnected"
+        @click="$router.push('edge-connect')"
+      />
+
+      <!-- test -->
+      <amb-button
         with-badge
         data-cy="heart"
+        is-icon
         icon="heart"
         :badge-content="newFavorites"
         :badge-value="newFavorites"
       />
 
-      <nav-button
+      <amb-button
```

Comment:
@vickywane why are you adding back the bell icon. It was commented out because it does not do anything meaningful.

Reply:
I think i started working on this feature when the icons were still in use.